package com.zelenux.upprpo_server_test.viewer.exceptions.groupExceptions;

public class GroupAlreadyHasUserException extends GroupException {
}
